### DISCUSSION TITLE
Lesson 4: "Could not find files for the given pattern(s).

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#199] [Q&A] [answered]
Halfway through Lesson 4 at the 3:58 mark, and when I hit run in VS Code I get the following output when generating the nonce:

Eric@ASUS MINGW64 ~/OneDrive/Arcus Cloud Consulting/Solidity/demos/web3_py_simple_storage
$ C:/Users/Eric/AppData/Local/Programs/Python/Python39/python.exe "c:/Users/Eric/OneDrive/Arcus Cloud Consulting/Solidity/demos/web3_py_simple_storage/deploy.py"
INFO: Could not find files for the given pattern(s).
0

I get the "0" as per the video, but there's also the "INFO: Could not find files for the given pattern(s).". I don't know why it's happening, and it doesn't seem to prevent me from running my code.

EDIT: I'm running a Windows 10 machine, but using bash as my terminal.

Thoughts?

**Comment by ArcusConsulting** [comment]
![Lesson4](https://user-images.githubusercontent.com/91853555/137414013-de9dc78f-6a97-4780-9116-74e7e45b4d83.jpg)

**Comment by PatrickAlphaC** [comment, marked as answer]
Please ignore this, it's just an INFO.

https://stackoverflow.com/a/68413900/11969592

**Reply by ArcusConsulting** [reply]
Thought so... Thanks!
